content/en/product_analytics/experimentation/defining_metrics.md

```diff
@@ -0,0 +1,107 @@
+</div>
+
+## Create your first metric
+
+To create a metric for your experiment: 
+
+1. Navigate to the [Metrics page][1] within Datadog Product Analytics.
+2. Click **+ Create Metric** at the top right corner. 
+3. Click **Select an Event** to see a list of all of the actions and views collected from the Datadog SDK.
+4. Add a metric name and optionally a description. Then, click **Save**.
+
+{{< img src="/product_analytics/experiment/exp_create_metric1.png" alt="UI page to create a metric." style="width:90%;" >}}
+
+## Specify metric aggregation
+
+### Default metric aggregation 
+After you’ve selected your event of interest, you can specify an aggregation method. Metrics default to a **count of unique** subjects (often users) that had at least one event. However, you can also choose to _count the total number_ of events, or _sum a property_ of that event:
+
+{{< img src="/product_analytics/experiment/exp_default_metric_agg.png" alt="Dropdown menu to select the method of aggregation for metrics." style="width:90%;" >}}
+
+
+### Default metric normalization 
+
+All metrics are normalized by the number of enrolled subjects. For example, a **count of unique users** metric is computed as: 
+
+```
+
+                      Number of users with specified event
+                  -----------------------------------------------           
+                      Number of users enrolled into this variant
+```
+
+Similarly, a **Sum of** metric is computed as: 
+
+
+```
+
+                  Sum of property over users enrolled in this variant
+              --------------------------------------------------------           
+                      Number of users enrolled into this variant
+```
+
+### Custom metric normalization
+
+You can also choose to normalize metrics by a different denominator. To do this, click on a metric
+then on **Create ratio**. This allows you to normalize your metric by another event, counting either the number of subjects with that event, the total number of events, or the sum of an event property. 
+
+{{< img src="/product_analytics/experiment/exp_create_ratio.png" alt="Button used to create a ratio from the metric." style="width:90%;" >}}
+
+For example, an e-commerce company that wants to measure the _Average Order Value_ can create a ratio metric with the sum of purchase revenue as the numerator, and the count of purchase events as the denominator. 
+
+Datadog’s statistical engine accounts for correlations between the numerator and denominator using the [delta method][2]. 
+
+
+## Add filters
+You can also add filters to your metrics, similar to other Product Analytics dashboards. For instance, you might want to filter page views based on referring URL or UTM parameters. Similarly, you might want to filter actions to a specific page or value of a custom attribute. As you add filters, you can check metric values in real time using the chart on the right.
```

Comment:
```suggestion
You can also add filters to your metrics, similar to other [Product Analytics dashboards][]. For instance, you might want to filter page views based on referring URL or UTM parameters. Similarly, you might want to filter actions to a specific page or value of a custom attribute. As you add filters, you can check metric values in real time using the chart on the right.
```
Not a blocker but it could be nice to link to Product analytics dashboard docs here ?